src/runtimes/docker/utils/normalizeContainerOS.ts

```diff
@@ -8,13 +8,13 @@
  * @param os pre-normalized OS name
  * @returns normalized OS name
  */
-export function getContainerOSFromPlatformOS(os?: string): ContainerOS {
-    switch (os || 'Linux') {
-        case 'Windows':
+export function normalizeContainerOS(os?: string): ContainerOS {
+    switch (os?.toLocaleLowerCase() || 'linux') {
```

Comment:
```suggestion
    switch (os?.toLowerCase() || 'linux') {
```
Don't actually want to do a locale-aware one here